app/src/main/java/com/alexis/myanimecompanion/data/remote/MyAnimeListAPI.kt

```diff
@@ -0,0 +1,66 @@
+package com.alexis.myanimecompanion.data.remote
+
+import com.alexis.myanimecompanion.QueryFieldsBuilder
+import com.alexis.myanimecompanion.data.remote.models.*
+import retrofit2.http.*
+
+interface MyAnimeListAPI {
+    @GET("anime")
+    suspend fun search(
+        @Query("q") query: String? = null,
```

Comment:
Change query parameter name to something more explicit

Reply:
Is it possible to remove default values? Is there any call to search() without all parameters? If not, please remove default values.

Reply:
I'll remote the default value, but I'm not sure about making the parameter name more explicit. What do you mean?

Reply:
I see. I was not sure q was a parameter from the API